reflect_test.go

```diff
@@ -82,12 +82,28 @@ type CustomTypeField struct {
 	CreatedAt CustomTime
 }
 
+type RootOneOf struct {
+	Field1 string      `json:"field1" jsonschema_oneof:"required=group1"`
```

Comment:
I like the approach, but I'd prefer the tag syntax to extend the existing `jsonschema` tag.

Reply:
Something like that?
jsonschema:’’oneof_type=string;array,oneof_required=group1’’   

Reply:
it will be more unreadable no ?

Reply:
btw, just spotted an error, i need to fix it